.appveyor.yml

```diff
@@ -1,36 +1,39 @@
-build: off
+build: false
 version: dev-{build}
 shallow_clone: false
 clone_folder: C:\projects\php-cs-fixer
 
 environment:
     matrix:
         - php_ver: 5.6.17
+          SKIP_LINT_TEST_CASES: 1
         - php_ver: 7.0.9
 
 cache:
-    - '%APPDATA%\Composer'
+    - '%LOCALAPPDATA%\Composer\files'
 
-install:
-    - choco install -my --allow-empty-checksums php -version %php_ver%
+init:
     - SET PATH=C:\tools\php;%PATH%
+
+install:
+    - choco install --yes --allow-empty-checksums php --version %php_ver%
     - cd C:\tools\php
     - copy php.ini-production php.ini
     - echo date.timezone="UTC" >> php.ini
     - echo memory_limit=512M >> php.ini
     - echo extension_dir=ext >> php.ini
     - echo extension=php_curl.dll >> php.ini
     - echo extension=php_openssl.dll >> php.ini
-    - cd C:\tools
-    - appveyor DownloadFile https://getcomposer.org/composer.phar
-    - appveyor DownloadFile https://phar.phpunit.de/phpunit-5.7.phar
+    - cd C:\projects\php-cs-fixer
+    - php -r "readfile('http://getcomposer.org/installer');" | php
```

Comment:
why changing the way of composer installation ?
I would either install it via choco (so we use AppVeyor choco cache) or keep phar download (so we could cache the phar file as well)